playbooks/technology-choices.md

```diff
@@ -6,106 +6,77 @@ description: Evaluating and adopting new technology at Artsy
-  [Cinder](https://github.com/artsy/cinder).
-- **CircleCI** for continuous integration over other services like Travis or Semaphore. See
-  [projects](https://circleci.com/build-insights/gh/artsy).
-- **JSON** over most other data formats.
-- **Kubernetes** over Heroku and AWS Opsworks for simple system deployments (or even some complex ones). We've
-  built out clusters, monitoring, and logging, and have solutions for common needs like data storage, CI, crons,
-  and shells. It's still the early days but production-ready. The [hokusai](https://github.com/artsy/hokusai)
-  utility embeds many of our adopted practices, or see [the production dash](https://kubernetes.artsy.net).
-- **Elasticsearch** for search, similarity, and sophisticated filtering. It replaces a large number of alternative
-  solutions including Solr, offline indexes, and data snapshots.
-- **Hypermedia** (usually [HAL](http://stateless.co/hal_specification.html)) is preferred over custom resource-ful
-  APIs. We like the self-documenting properties of a HAL API as well as how easily it can be consumed by clients
-  (e.g., via [`hyperclient`](<(https://github.com/codegram/hyperclient)>). There remain concerns about clients
-  being able to efficiently traverse large sets or extended relations, so these days for most purposes we prefer:
-- **GraphQL** is our go-to option for API communication, popular for the focused, customizable responses and direct
-  applicability to client views. There are concerns include efficiency.
-- **RabbitMQ** offers a lightweight alternative to choosing _any_ API between systems. Its stream allows source
-  systems to publish their notable events and other systems to subscribe as desired. The events themselves tend to
-  be minimal JSON blobs, so some other data-sharing (e.g., via API) may be required. We prefer joining systems in
-  this event-oriented way over directly invoking downstream systems from upstream ones.
-
-### The Artsy Ecosystem
-
-- **Focused services** over monolithic projects to ease developers' cognitive overhead, speed development and
-  testing, and allow for flexible technology evolution (see above). Projects can be focused around a _function_
-  (such as messaging) or a _domain_ (such as consignments).
-- **Focused, smaller management utilities** over earlier iterations of admin utilities such as
-  [Torque](https://admin.artsy.net). Examples include [CMS](https://cms.artsy.net)
-  ([Volt](https://github.com/artsy/volt)) for partners, [Helix](https://helix.artsy.net) for genomers, and
-  [auctions.artsy.net](https://auctions.artsy.net) ([Ohm](https://github.com/artsy/ohm)) for managing sales. We aim
-  to _opportunitistically_ transition remaining administrative functions (such as for inquiries or users) to new
-  tools.
-- **Opaque IDs** over "slugs" for any programmatic access. While slugs are more friendly for URLs, they can change
```

Comment:
This is interesting as it is something that doesn't totally fit into the model of a radar (or perhaps I don't see it).

How do you think we should communicate out these technology strategies given the approach of this PR? Shall we migrate to the radar?

Reply:
Yeah the radar's quadrants aren't always a good fit. I think it's still better to cram it into a category that might not be perfect anyway. At least it's documented and discussed that way. Will add this.